stories/atoms/components/listItem/ListItem.stories.mdx

```diff
@@ -107,26 +107,21 @@ The `description` property allows you to provide additional information. It can
 <Canvas>
   <Story name="Description">
     {() => (
-      <List
-        items={[
-          {
-            title: 'Item 1',
-            description: 'The additional information of the Item 1'
-          },
-          {
-            title: 'Item 2',
-            description: (
-              <div>
-                <Typography as="div" color="text" fontSize="small">
-                  Additional information in Item 2 that may be more complex than a simple `string`.
-                  The description can display any content.
-                </Typography>
-                <ProgressBar label="okp4.pdf" minValue={0} maxValue={195} currentValue={23} />
-              </div>
-            )
+      <List>
```

Comment:
And you also can remove this part of code: 
` items: {
      table: {
        disable: true
      }
    }` from the `argTypes`